docs/DeveloperGenerateTest.md

```diff
@@ -50,18 +75,41 @@ var _ = framework.ChaosSuiteDescribe("Chaos tests", Label("Chaos"), func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	Describe("Categorizing book length ", Label("book"), func() {
+		// Declare variables here.
+
+		When("the book has more than 300 pages ", Label("slow"), func() {
+
+			It("Should be a novel", func() {
```

Comment:
I think in `It` specs, it is a good practice to start with lowercase

Reply:
@rsoaresd  Good point! But this can be controlled when writing the actual text outline. So when I wrote the text outline file I started the description text with a capital letter by accident. 
https://github.com/redhat-appstudio/e2e-tests/blob/61ccfe3449dfa3bb6b7c420958f13664ba0bb7db/docs/DeveloperGenerateTest.md?plain=1#L11

So If we write that description text with lowercase it will be rendered in lowercase. Do you think when we build the graph from the text outline we should lowercase the description text for all It containers?

Reply:
@Dannyb48, I do not think it is strictly necessary. However, I think it makes sense to follow the good practice